packages/stencil/components/switch/src/components/osds-switch/styles/osds-switch.mixins.scss

```diff
@@ -0,0 +1,14 @@
+// general mixins and functions for the component
+
+@mixin osds-switch-on-selected-host() {
+  :host(&) {
+    @content;
+  }
+}
+
+@mixin osds-switch-on-switch-items-child($property: '') {
```

Comment:
seems weird to style from the parent
each child should contains its own style regarding of its state, which is passed from the parent